float64 leg_pos_kp
float64 leg_pos_kd
float64 hip_pos_kp
float64 hip_pos_kd
float64 left_toe_pos
float64 right_toe_pos
int8    main_controller
float64 maxCurrent
float64 maxSpeed
float64 maxDefl
float64 maxTauRate

# These allow the GUI to control tau
int8    tauMode
float64 manualTau
